content/english/security/1- Meeting Alice, Bob and Eve/_index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Meeting Alice, Bob and Eve"
+description: ""
+draft: true
+weight: 1
```

Comment:
Consider prereq & difficulty like other workshops :)

Reply:
Is it alright if we have it on the table of contents page?